konfera/models/ticket.py

```diff
@@ -58,4 +58,12 @@ def save(self, *args, **kwargs):
             order = Order(price=self.type.price, discount=discount, status=AWAITING, purchase_date=timezone.now())
             order.save()
             self.order = order
-        super(Ticket, self).save(*args, **kwargs)
+            super(Ticket, self).save(*args, **kwargs)
+        else:
+            super(Ticket, self).save(*args, **kwargs)
```

Comment:
I see your point.

The original issue #149 was suggesting that login would be in order. I think it is better solution to have `recalculate_ticket_price` (or similar name) function in order. This function would the logic you have below.

We can update order save method to check for flag `_recalculate_tickets` and if True it would call the function `recalculate_ticket_price`. This way when creating new order (doesnt matter if from model or admin) we dont have to do the search for tickets and in ticket you can be sure that you update the price only for existing tickets (after save). I think we would end up with one extra if (but it seems easier to understand from code rather than have two super), and the recalculate_ticket_price will be in order where it makes more sense.